SPECS/ruby/ruby.spec

```diff
@@ -97,6 +97,7 @@ Source4:        rubygems.con
 Source5:        rubygems.prov
 Source6:        rubygems.req
 Source7:        macros.rubygems
+Patch0:         CVE-2023-36617.patch
```

Comment:
Could you add a note here why this patch was needed, and if needs to be modified when ruby/rubygem-uri is updated in the future, thanks

Reply:
Added comment to patch and SPEC file. Thanks!